README.md

```diff
@@ -0,0 +1,7 @@
+
+
```

Comment:
@swalladge Thanks for getting us started with a README here :)

Could you remove these extra blank lines when you get a minute?